client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -125,23 +126,33 @@ class StatsPostDetail extends Component {
 			noViewsLabel = translate( 'Your post has not received any views yet!' );
 		}
 
+		// Set up for FixedNavigationHeader.
+		const domain = this.props.context?.path.split( '/' ).pop();
+		const backLabel = translate( 'Traffic' );
+		const backLink = '/stats/day/' + ( domain || '' );
+		const navigationItems = [ { label: backLabel, href: backLink }, { label: title } ];
+		const showHeaderCake = false;
+
 		return (
-			<Main wideLayout>
+			<Main className="has-fixed-nav" wideLayout>
 				<PageViewTracker
 					path={ `/stats/${ postType }/:post_id/:site` }
 					title={ `Stats > Single ${ titlecase( postType ) }` }
 				/>
 				{ siteId && ! isLatestPostsHomepage && <QueryPosts siteId={ siteId } postId={ postId } /> }
 				{ siteId && <QueryPostStats siteId={ siteId } postId={ postId } /> }
 
-				<HeaderCake
-					onClick={ this.goBack }
-					actionIcon={ showViewLink ? 'visible' : null }
-					actionText={ showViewLink ? actionLabel : null }
-					actionOnClick={ showViewLink ? this.openPreview : null }
-				>
-					{ title }
-				</HeaderCake>
+				<FixedNavigationHeader navigationItems={ navigationItems } />
+				{ showHeaderCake && (
```

Comment:
Feature flags were made for this exact use case 😁 

You can define new feature flags in one of the environment JSONs [here](https://github.com/Automattic/wp-calypso/tree/trunk/config). I'd recommend adding one to the [development.json](https://github.com/Automattic/wp-calypso/blob/trunk/config/development.json#L28), which will automatically enable the feature only for local Calypso instances.

Afterward, you can do the following:

```js
// 'stats/new-header' is the new feature flag name
config.isEnabled( 'stats/new-header' ) ? <FixedNavigationHeader {...} /> : <HeaderCake {...} />
```

Note that we'll want to conditionally apply the `has-fixed-nav` class name to the Main component if we go this route.